NuGet.config

```diff
@@ -42,6 +43,9 @@
     <packageSource key="dotnet-eng">
       <package pattern="*" />
     </packageSource>
+    <packageSource key="nuget">
+      <package pattern="Microsoft.DevTunnels.*" />
```

Comment:
@DamianEdwards do you still need these changes?

Reply:
nope, will revert